modules/caddytls/values.go

```diff
@@ -127,9 +128,35 @@ var SupportedProtocols = map[string]uint16{
 	"tls1.3": tls.VersionTLS13,
 }
 
+// UnsupportedProtocols is a map of unsupported protocols.
```

Comment:
It might be worth noting in the godoc that this is just for logging purposes, not enforcement.

Reply:
Pushed a commit to update the godoc and not export `unsupportedProtocols`.  Let me know if you agree with the change.